WEAPON/CreateWeaponObject.md

```diff
@@ -8,23 +8,26 @@ ns: WEAPON
 Object CREATE_WEAPON_OBJECT(Hash weaponHash, int ammoCount, float x, float y, float z, BOOL showWorldModel, float scale, Any p7);
 ```
 
+Creates a weapon object in the world at the specified location.
+
 ```
 Now has 8 params.  
 ```
 
+**Note** To create a networked weapon object and attach it to the player, use [CREATE_OBJECT](#_0x509D5878EB39E842) instead.
+For more details see [Weapon objects poor sync](https://github.com/citizenfx/fivem/issues/2249).
```

Comment:
I don't think it's relevant to use a Github issue for reference in a native description..